config.go

```diff
@@ -331,6 +331,17 @@ func (b *Builder) SetUser(spec string) {
 	b.Docker.Config.User = spec
 }
 
+// OnBuild returns the OnBuild value from the container.
+func (b *Builder) GetOnBuild() []string {
+	return b.Docker.Config.OnBuild
```

Comment:
Please rename this to `OnBuild()`, and use `copyStringSlice()` to make a copy of the value to return.  We don't want people to think that modifying the slice that they get back is a supported way of changing the value we're holding.

Reply:
fixed (another missing comment)